homework/index2.js

```diff
@@ -0,0 +1,44 @@
+function getPromise() {
```

Comment:
Avoid unused files. If you want to test some crazy stuff, better to do that on a separate branch; you can create as many experimental branches as you want. But don't pollute your main branch with it.